src/components/view/View.tsx

```diff
@@ -309,6 +309,8 @@ class View<NodeType, EdgeType> extends Preact.Component<ViewProps<NodeType, Edge
       scale: workspaceScale
     } = this.props.state.workspace;
 
+    const overflow = this.props.configService.getViewMode() ===  'Scroll' ? 'scroll' : 'hidden';
```

Comment:
I was hoping that in scroll mode, we will use diagram maker workspace pan actions to scroll the view instead of the browser scroll. That would ensure that diagram maker retains the control of the DOM positions instead of giving that back to the browser.

Let me know if you see issues with that approach.